CONTRIBUTING.md

```diff
@@ -1,17 +1,183 @@
-# Contributing to LLVM
+# Contributing to the Arm Toolchain
 
-Thank you for your interest in contributing to LLVM! There are many ways to
-contribute, and we appreciate all contributions.
+The Arm Toolchain repository is a fork of the *llvm-project*, with
+build scripts to produce Arm Toolchain for Embedded and Arm Toolchain
+for Linux.
 
-To get started with contributing, please take a look at the
-[Contributing to LLVM](https://llvm.org/docs/Contributing.html) guide. It
-describes how to get involved, raise issues and submit patches.
+Arm Toolchain for Embedded integrates *llvm-project* with the
+*picolibc* C-library or the *newlib* C-library via an overlay package.
 
-## Getting in touch
+Contributions are welcome that improve the project in areas including
+but not limited to:
+* Integration of upstream components.
+* Packaging.
+* Testing of the above.
+* Documentation.
 
-Join the [LLVM Discourse forums](https://discourse.llvm.org/), [Discord
-chat](https://discord.gg/xS7Z362), or #llvm IRC channel on
-[OFTC](https://oftc.net/).
+The Arm Toolchain repository is synchronized with the upstream
+*llvm-project* including the main and release branches. Arm Toolchain
+for Embedded aims to stay close to the head of the *picolibc* main
+branch. The *newlib* overlay package aims to track the most recent
+*newlib* release.
 
-The LLVM project has adopted a [code of conduct](https://llvm.org/docs/CodeOfConduct.html) for
-participants to all modes of communication within the project.
+All changes that can be made in the upstream project, must be made in
+the upstream project. Please see the [Downstream Patch Policy](#downstream-patch-policy)
+below for what Arm considers a downstream patch along with the
+additional requirements for such a patch.
+
+For guidance on how to contribute to the upstream projects see:
+* *llvm-project* [Contributing to LLVM](https://llvm.org/docs/Contributing.html).
+* *picolibc* GitHub.
+[Pull requests documentation](https://docs.github.com/en/pull-requests)
+
+## Ways to contribute
+
+### Report an issue
+
+Please make sure that the issue is specific to the *arm-toolchain*
+before reporting. If the issue exists in *llvm-project* or *picolibc*
+then please report the issue in the upstream project.
+
+Please create a GitHub issue in the *arm-toolchain* project
+[Issues](https://github.com/arm/arm-toolchain/issues)
+list.
+
+### Submit a fix
+
+For a small change, please create a Pull Request as described in
+_How to submit a change_ section below.
+
+### Suggest a feature or bigger change
+
+For a bigger change, please create an issue in the
+*arm-toolchain* project
+[Issues](https://github.com/arm/arm-toolchain/issues)
+list and put [RFC] in the subject line (Request for Comments) to initiate the discussion
+first, before submitting a PR for the change.
+
+There is no formal template for an `rfc`, however it would be good to
```

Comment:
`rfc` was probably a label previously, so may be clearer as RFC now with the new recommendation.

Reply:
ACK